### DISCUSSION TITLE
Versioned docs pointing to updated content

### DISCUSSION THREAD

**Discussion by ofek** [squidfunk/mkdocs-material#3746] [Q&A]
Is there a way for a versioned page to always display the latest content of its source? My use case is for a list of downstream users, and possibly the changelog.

**Comment by squidfunk** [comment]
I'm not sure I understand the question. Could you please provide some more information and an example?

**Reply by ofek** [reply]
Say /users is rendered from Git /docs/users.md, I want /vX.Y.Z/users to always show what's on master or redirect to/display /dev/users that is always built from master.

**Reply by squidfunk** [reply]
I see. I suggest that you ask this on the `mike` repository because it's not exclusive to Material for MkDocs.

**Reply by ofek** [reply]
https://github.com/jimporter/mike/issues/97

**Reply by ofek** [reply]
I'm not great with HTML, is something like this possible?

```
# Users

-----

## Organizations

<inject-on-load="../organizations-table.html">

## Projects

<inject-on-load="../projects-table.html">
```

**Reply by squidfunk** [reply]
Using an `iframe`, yes, but that might bring other challenges.
